agent/engine/task_manager.go

```diff
@@ -360,7 +374,6 @@ func (mtask *managedTask) waitEvent(stopWaiting <-chan struct{}) bool {
 		mtask.handleResourceStateChange(resChange)
 		return false
 	case <-stopWaiting:
-		seelog.Infof("Managed task [%s]: no longer waiting", mtask.Arn)
```

Comment:
why is this removed?

Reply:
In the case that we are not exiting, this will get logged here instead: https://github.com/aws/amazon-ecs-agent/pull/2462/files#diff-20346f7af6202828de353f0137b6e43aR330